File: api/v2010/call/create-example-9/create-example-9.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

call = @client.calls.create(
                       sip_auth_password: 'secret',
                       sip_auth_username: 'jack',
                       url: 'http://www.example.com/sipdial.xml',
                       to: 'sip:kate@example.com',
                       from: 'Jack'
                     )

puts call.sid

File: api/v2010/call/create-example-9/create-example-9.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        // DANGER! This is insecure. See http://twil.io/secure
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var call = CallResource.Create(
            sipAuthPassword: "secret",
            sipAuthUsername: "jack",
            url: new Uri("http://www.example.com/sipdial.xml"),
            to: new Twilio.Types.PhoneNumber("sip:kate@example.com"),
            from: new Twilio.Types.PhoneNumber("Jack")
        );

        Console.WriteLine(call.Sid);
    }
}
